docs/source/checks/simple.rst

```diff
@@ -0,0 +1,175 @@
+.. _simple:
+
+Simple Check
+====================
+
+Description
+-----------
+
+The SimpleCheck gives you basic information about your datasets and detects the most obvious of datashifts.
+
+Some shifts result in the change of basic metrics, that are commonly used for the analysis and exploration of datasets.
+Those metrics and statistics are calculated on both datasets and compared afterwards. HIER NOCH AMAZON DEEQU ERWÄHNEN,
+WIRD NACH UMBENENNUNG DES CHECKS GEMACHT
+
+The basic check should be used in most analyses because its metrics are scientifically grounded and commmonly used.
+It works directly on categorical and numerical data and, with additional Precalculations, also on text data.
+
+
+Example
+-------
+
+::
+
+    from shift_detector.detector import Detector
+    from shift_detector.checks.simple_check import SimpleCheck
+
+    data_set_1 = 'examples/pokedex1.csv'
+    data_set_2 = 'examples/pokedex2.csv'
+
+    detector = Detector(
+        data_set_1,
+        data_set_2,
+        delimiter=','
+    )
+
+    detector.run(SimpleCheck())
+    detector.evaluate()
+
+The code works as follows:
+
+1. First, you create a :class:`~shift_detector.detector.Detector` object to tell Morpheus
+   which data sets you want to compare.
+2. Then you start the detector with
+   :meth:`~shift_detector.detector.Detector.run` and the checks you want to run: in this case
+   :class:`~shift_detector.checks.simple_check.SimpleCheck`.
+3. Finally, you print the result with
+   :meth:`~shift_detector.detector.Detector.evaluate`
+
+Result
+++++++
+
+The SimpleCheck produces this report:
+
+::
+
+    Numerical Columns:
+
+    Column 'payment':
+    Metric: mean with Diff: -29.4 %, threshold: 20%
+    Metric: std with Diff: +80.2 %, threshold: 50%
+
+    (...)
+
+
+    Categorical Columns
+
+    Column 'payment_option':
+    Attribute: 'debit' with Diff: +6.0 %, categorical threshold: 5%
+    Attribute: 'cash' with Diff: -6.5 %, categorical threshold: 5%
+
+    (...)
+
+
+
+Interpretation
+++++++++++++++
+
+The results show column-wise wich shifts appear in your dataset. First the reports of your numerical columns are shown,
+after that the reports of your categorical columns.
```

Comment:
What about textual columns? You wrote about them earlier

Reply:
youre right. see #127